### ISSUE TITLE
Compact links encoding in transactions

### ISSUE THREAD

**Issue by arturalbov** [cybercongress/cyberd#212]
### User Story

Size of transactions already reduced by #171 . We should think about more compact links encoding to reduce tx size even more.

For example, cidFrom[cidTo1,cidTo2,cidTo3] instead of cidFromCidTo1cidFromCidTo2cidFromCidTo3. Some binary encoded compact structure in the link message.

[comment]: # (As a <user type>, I want to <task> so that <goal>.)

### Why Is this Needed?

Reduce storage size and network load.

[comment]: # (Describe the problem and why this task is needed. Provide description of the current state, what you would like to happen, and what actually happen)

**Comment by xhipster** [comment]
It is a good thing to change in the begining

**Comment by hleb-albau** [comment]
@xhipster Not really, currently we made 90% of possible reduction just by small changes. But this one requires a lot of extra research, so for now, it postponed.

**Comment by hleb-albau** [comment]
From @xhipster discussion.
See https://medium.com/@o_van_wijk/tangle-pathway-58fb13e19eff.
